Decent.Minecraft.Client/RailType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Decent.Minecraft.Client
+{
+    /// <summary>
+    /// The type of rail.
+    /// </summary>
+    public enum RailType : byte
```

Comment:
I disagree, direction is clearly a property. This literally is the type of rail, so why isn't it a type of rail :) ?